lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -35,22 +35,22 @@ sub edit_user : Path('/admin/user/edit') Args(1) RequireAuth HiddenOnSlaves
     my $form2 = $c->form(
         form => 'User::EditProfile',
         item => {
-            email            => $user->email,
+            username          => $user->name,
+            email             => $user->email,
             skip_verification => 0,
-            website            => $user->website,
-            biography        => $user->biography
+            website           => $user->website,
+            biography         => $user->biography
```

Comment:
I wouldn’t bother with this optical alignment stuff. If you do, make it a separate commit please.